olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogClient.h

```diff
@@ -44,10 +44,30 @@ namespace read {
 class CatalogClientImpl;
 
 /**
- * @brief The CatalogClient class. Marshals Requests and their Results.
+ * @brief The `CatalogClient` class marshals `OLP` requests and their results.
```

Comment:
Question. Do we call remote instances as `OLP` or we refer to the APIs by their respective names, e.g. `Data API`?
For example, in this case, would it make sense to say?
```
The `CatalogClient` class marshals requests to the `Data API` and their results.
```

Reply:
To be more explicit we should refer to the APIs.